### PR TITLE
EDITOR and CORRESPONDENT types are exported to NewsML as AUTHOR

### PR DESCRIPTION
**Pull request by ride90**
SDBELGA-401

### CODE REVIEW

File: server/belga/publish/belga_newsml_1_2.py [hunk 1]
```diff
@@ -1040,42 +1040,32 @@ def _format_newscomponent_3_level(self, newscomponent_2_level, item):
...
-            'initials': '',
-            'role': ''
-        }
+        author_info = {'initials': '', 'role': ''}
 
-        # get author_id
-        if type(author) is str:
+        if type(author) is dict:
+            author_info['initials'] = author.get(
+                'sub_label',
```
Comment (idsvn) [inline server/belga/publish/belga_newsml_1_2.py:1047]: Based on `SDBELGA-278`, `initials` should be username, but `sub_label` is user full name, isn't it?

Comment (ride90) [reply server/belga/publish/belga_newsml_1_2.py:1047]: indeed, thanks

**Comment by ride90** [comment]
@idsvn , please, check it ⬆️

**Comment by ride90** [comment]
CI is dead. Sending another PR to fix it, after will merge this one.

**Pull request merged**